### ISSUE TITLE
TypeError [ERR_INVALID_URL]: Invalid URL when sourcemap contains invalid paths

### ISSUE THREAD

**Issue by alan-agius4** [ampproject/remapping#157]
## Current
When the sourcemap looks like (which is invalid) @ampproject/remapping will throw an `ERR_INVALID_URL` exception.  

```
{"version":3,"sources":["webpack://[name]/webpack/universalModuleDefinition", ...rest], ...rest}
```

<pre><code>./node_modules/@clickup/gantt/codebase/dhtmlxgantt.js - Error: Module build failed (from ./node_modules/@angular-devkit/build-angular/src/babel/webpack-loader.js):
TypeError [ERR_INVALID_URL]: Invalid URL
    at new NodeError (node:internal/errors:371:5)
    at onParseError (node:internal/url:552:9)
    at new URL (node:internal/url:628:5)
    at normalizeSimplePath (/Users/mattlewis/Code/open-source/source-map-bug/node_modules/@ampproject/remapping/dist/remapping.umd.js:301:26)
    at normalizePath (/Users/mattlewis/Code/open-source/source-map-bug/node_modules/@ampproject/remapping/dist/remapping.umd.js:320:20)
    at resolve (/Users/mattlewis/Code/open-source/source-map-bug/node_modules/@ampproject/remapping/dist/remapping.umd.js:386:26)
    at resolve$1 (/Users/mattlewis/Code/open-source/source-map-bug/node_modules/@ampproject/remapping/dist/remapping.umd.js:418:16)
    at /Users/mattlewis/Code/open-source/source-map-bug/node_modules/@ampproject/remapping/dist/remapping.umd.js:737:25
    at Array.map (<anonymous>)
    at buildSourceMapTree (/Users/mattlewis/Code/open-source/source-map-bug/node_modules/@ampproject/remapping/dist/remapping.umd.js:734:34)
</code></pre>

## Expected
Gracefully handle invalid sourcemaps.

## Other
Context https://github.com/angular/angular-cli/issues/22526

**Comment by jridgewell** [comment]
Closed with 3a7e32fad4ec5028e5db7d148854eaa3f62a289f, released in v1.1.0.

**Comment by alan-agius4** [comment]
Thanks @jridgewell
